setup.cfg

```diff
@@ -32,6 +32,7 @@ package_dir =
 python_requires = >=3.7
 
 install_requires =
+    ipython<8.0
```

Comment:
Why is this required? I would prefer not to pin to old versions of any more libraries if at all possible.

Reply:
There was an error during the test caused by the newest IPython version. Therefore I added this requirement.

Reply:
I have also seen some recent incompatibility issues with ipython, makes sense

Reply:
Maybe in the new release of IPython 8.13.2 this issue was resolved.

Reply:
Thanks!